package/common_init/overlay/etc/init.d/copy_duro_eeprom.sh

```diff
@@ -41,6 +42,16 @@ copy_duro_eeprom()
   chmod 0644 "$DURO_EEPROM_TMP_PATH"
   logi "Moving $DURO_EEPROM_TMP_PATH to $DURO_EEPROM_CFG_PATH."
   mv $DURO_EEPROM_TMP_PATH $DURO_EEPROM_CFG_PATH
+
+  # Parse eeprom and set is_duro flag if string "duro" appears in eeprom.
+  # Downstream systems can either read EEPROM contents or check is_duro flag.
+  # pfwp uses is_duro flag as an IPC method to enable can termination setting
+
+  if grep -q "^${DEVICE_DURO_ID_STRING}" $DURO_EEPROM_CFG_PATH; then
+    echo "1" > /etc/flags/is_duro
+  else
+    echo "0" > /etc/flags/is_duro
```

Comment:
In other implementations of this pattern, there is an overlay that guarantees the file is created regardless of script failure/other corner cases. See [here](https://github.com/swift-nav/piksi_buildroot/blob/master/package/starling_daemon/overlay/etc/flags/starling_on_linux) for example. Can we add one of these, maybe` common_init` is the most appropriate place.